tests/conftest.py

```diff
@@ -10,15 +10,16 @@
 from httpx import AsyncClient
 
 from lnbits.app import create_app
-from lnbits.core.crud import create_account, create_wallet
+from lnbits.core.crud import create_account, create_wallet, get_user
 from lnbits.core.models import CreateInvoice
 from lnbits.core.services import update_wallet_balance
 from lnbits.core.views.api import api_payments_create_invoice
 from lnbits.db import Database
 from lnbits.settings import settings
 from tests.helpers import get_hold_invoice, get_random_invoice_data, get_real_invoice
 
-# dont install extensions for tests
+# override settings for tests
+settings.lnbits_admin_ui = True
```

Comment:
i guess this could lead to uncomfortable situations in local testing, where the tests now use values from your db while you'd like the tests to use your env

Reply:
no, i think that i compleltly fine, when using `pytest` without `make test` we are lacking those `.env` we set in the make command. so i actually think conftest is the place to put them!